mlflow/langchain/__init__.py

```diff
@@ -84,6 +86,17 @@
 
 FLAVOR_NAME = "langchain"
 _MODEL_TYPE_KEY = "model_type"
+_DATABRICKS_RAG_CONFIG = "databricks_rag_config"
+__chain__ = None
+__config_path__ = None
+
+
+def _set_chain(chain):
+    globals()["__chain__"] = chain
+
+
+def _set_config_path(path):
+    globals()["__config_path__"] = path
```

Comment:
Should this be defined in `databricks.rag` instead?

Reply:
We need that users in MLflow folks and it is very weird that mlflow calls databricks.rag and create a cyclic dependency. So databricks.rag imports __config_path__ to get that information. 

Also made it more generic. Let me know if that makes sense. :)